src/libraries/Microsoft.PowerFx.Interpreter/Environment/PowerFxConfigExtensions.cs

```diff
@@ -80,6 +80,12 @@ public static void EnableRegExFunctions(this PowerFxConfig config, TimeSpan regE
             }
         }
 
+        [Obsolete("Join is still in preview.")]
```

Comment:
> [Obsolete("Join is still in preview.")] [](http://example.com/codeflow?start=7&length=40)

Greg & I chatted and are good with this. 
It will let us test in playground and DV.